test/pack/pack.c

```diff
@@ -390,7 +395,11 @@ void *runtest(void *arg)
                 break;
 
             case YAKSA_OP__PROD:
-                rc = DTP_obj_buf_check(dobj, dbuf_h, 0, -1, basecount);
+                if (strstr(typestr, "complex") != NULL) {
+                    rc = DTP_obj_buf_check(dobj, dbuf_h, 0, 0, basecount);
+                } else {
+                    rc = DTP_obj_buf_check(dobj, dbuf_h, 0, -1, basecount);
```

Comment:
Just for my education, can you explain how the buffer is computed?

Reply:
I cherry-picked this one from PR #177. @zhenggb72 - Can you please explain?

Reply:
Originally the two buffers are initialized with 0, 1, 2, 3 ...  and  -1, -1 ..... And the result is  0, -1, -2, -3, -4 ...

For complex case, each complex takes two numbers  real part and  img part. so the result of product is not as simple as 0, -1, -2. So it is wrong.
This commit tried to fix it in a very simple way, just to make it work.

Reply:
I see. Could you add a comment at line 226 -- "Multiply 0 since there is no easy way to verify complex multiplication"?